cirq-superstaq/cirq_superstaq/job.py

```diff
@@ -164,6 +164,17 @@ def compiled_circuit(self) -> cirq.Circuit:
 
         return css.deserialize_circuits(self._job["compiled_circuit"])[0]
 
+    def input_circuit(self) -> cirq.Circuit:
+        """Gets the input circuit that was submitted for this job.
+
+        Returns:
+            The pre-compile circuit.
```

Comment:
```suggestion
            The pre-compiled circuit.
```
or `pre-compilation`? or just `input`? I'm not sure, but pre-compile sounds weird to me.

Reply:
That is a fair point, I will use "input" as the standard between css and qss.